cirq-google/cirq_google/engine/processor_sampler.py

```diff
@@ -31,17 +32,21 @@ def __init__(self, *, processor: 'cg.engine.AbstractProcessor'):
         """
         self._processor = processor
 
-    def run_sweep(
+    async def run_sweep_async(
```

Comment:
While we're here, these methods also need `Args:` and `Returns:` in their docstrings.

Reply:
In the generated docs it looks like this just inherits the docstring from the abstract method in the parent class: https://quantumai.google/reference/python/cirq_google/engine/ProcessorSampler#run_sweep; ipython/colab handle this similarly when getting help with `cirq_google.ProcessorSampler.run_sweep?` by showing the parent class docstring. I'd prefer to rely on that rather than duplicating the docs here in code, where they are bound to drift apart over time.

Reply:
SGTM.